spec/classes/init_spec.rb

```diff
@@ -473,6 +473,8 @@
     it { should contain_file('sshd_config').with_content(/^HostKey \/etc\/ssh\/ssh_host_rsa_key/) }
     it { should contain_file('sshd_config').with_content(/^HostKey \/etc\/ssh\/ssh_host_dsa_key/) }
     it { should contain_file('sshd_config').with_content(/^StrictModes yes$/) }
+    it { should_not contain_file('sshd_config').with_content(/^PermitUserEnvironment/) }
```

Comment:
Are you saying I should have an entry with the default value in each OS block from lines 25-185?  The current implementation leaves these two options commented out, so I didn't want to add something to the configuration unless it was explicitly stated.

Would adding default values also mean updating the manifest to set the default values?

Do you have a good resource for finding out what the default values are?